apps/sensenet/src/components/dialogs/upload/upload-dialog.tsx

```diff
@@ -211,7 +211,10 @@ export function UploadDialog(props: UploadDialogProps) {
         </Grid>
       </DialogContent>
       <input
-        onChange={ev => ev.target.files && addFiles([...ev.target.files])}
+        onChange={ev => {
+          ev.target.files && addFiles([...ev.target.files])
+          ev.target.value = ''
```

Comment:
I would rather see this fix in the removeItem method. Set `inputFile` value field and add a comment please that states why was this necessary.